docs/user_guide/static_assets.md

```diff
@@ -0,0 +1,128 @@
+# Add custom CSS and JS assets
+
+If you'd like to modify this theme or sections on the page, you'll need to add custom CSS or JavaScript to your theme.
+Since this is a common operation we cover a few ways to do this here.
+
+````{admonition} Sample site structure
+In all examples below, assume we have a site structure like this:
+
+```
+mysphinxsite/
+├── _static
+│   ├── mycss.css
+│   └── myjs.js
+└── conf.py
+```
+````
+
+## Define a list of asset folders in `conf.py`
+
+The simplest way to add JS and CSS assets is to use [`html_css_files`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_css_files) and [`html_js_files`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_js_files) in your `conf.py` file.
+Each can be a list of paths, _relative to your `html_static_path`_.
+They will be added to the `<head>` of your site in order.
+
+For example:
+
+```{code-block} python
+:caption: conf.py
+
+html_static_path = ["_static"]
```

Comment:
This is awesome. where does it add the files in the head? will it add them towards the end of is there some specific section?

many many thanks and so sorry for nerd sniping goodness.

Reply:
good question, I'm not quite sure but I think it adds them at the end

Reply:
it add them at the end (we made sure to add custom files after everything related to the theme) but the html_static_path is doing much more. Everything in this will be copied to the build directory even if it's never used

Reply:
ah good point @12rambau - I added a little description of `html_static_folder` at the top.

Reply:
thanks!! you both rock!